independent-publisher-connectors/SMS Wireless Services/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@SietseBakkerViaData May I ask you please to fix the indentation for all your files. all files should be properly formatted.
Thank you so much.